geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -408,7 +430,7 @@ mod tests {
         );
 
         // Now try a (dummy) reload, replacing TestPlugin with TestPlugin2
-        let reload_result = plugin_manager_lock.reload_plugin(DUMMY_NAME, DUMMY_CONFIG);
+        let reload_result = plugin_manager_lock.reload_plugin(DUMMY_NAME, TESTPLUGIN2_CONFIG);
```

Comment:
Would it be worth it to extend this test to demonstrate that a subsequent reload would need to pass the new name `ANOTHER_DUMMY_NAME` as the first parameter? I just want to make sure the behavior is clear.

Reply:
I think in here the intention was to reload the same named "DUMMY_NAME" plugin with a different configuration file ( and probably different .so file) which trigger unload of the old library and loading new library. During the reload the name should be the same. If it is a different plugin name, it will be a new "load" not reload.

Reply:
I see your point. But I do not see the need to enforce the rule of the reload name and the loaded plugin to have the name as that critical -- maybe it is more flexible with the current implementation. I can extend the test.